### PR TITLE
refactor whitelist

### PR DESCRIPTION
**Pull request by yubrew**

### CODE REVIEW

File: contracts/minter/src/contract.rs [hunk 1]
```diff
@@ -522,13 +508,19 @@ pub fn mint_price(deps: Deps, admin_no_fee: bool) -> Result<Coin, StdError> {
...
+}
+
 #[cfg_attr(not(feature = "library"), entry_point)]
 pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> StdResult<Binary> {
     match msg {
         QueryMsg::Config {} => to_binary(&query_config(deps)?),
         QueryMsg::StartTime {} => to_binary(&query_start_time(deps)?),
         QueryMsg::MintableNumTokens {} => to_binary(&query_mintable_num_tokens(deps)?),
         QueryMsg::MintPrice {} => to_binary(&query_mint_price(deps)?),
+        QueryMsg::MinterAddresses {} => to_binary(&query_minter_addresses(deps)?),
```
Comment (shanev) [inline contracts/minter/src/contract.rs:523]: Would this ever need be queried? Isn't it more for internal accounting?

**Review by shanev — CHANGES_REQUESTED** [review]

File: contracts/minter/src/contract.rs (hunk 1 above)
Comment (yubrew) [reply contracts/minter/src/contract.rs:523]: Initially thinking creators might want to know distribution of addresses doing the minting. Now that I'm thinking about it, seems like something no one asked for and few people really want. Removing this bloat.

**Comment by JakeHartnell** [comment]
Needs `cargo schema` update.

**Review by shanev — APPROVED** [review]
LGTM

**Comment by yubrew** [comment]
updated schema

**Comment by jhernandezb** [comment]
Can we add a MintCount query? this will make the airdrop claimer easier

**Review by JakeHartnell — APPROVED** [review]
+1 on @jhernandezb's request but can be a separate PR IMO.

**Comment by shanev** [comment]
Created #110

**Pull request merged**